### ISSUE TITLE
Wear app

### ISSUE THREAD

**Issue by yschimke** [waseefakhtar/dose-android#108]
Wondering if you would accept a Wear app as a contribution? Probably just working over the Data layer to communication from Watch to Mobile.

I work on Google Wear OS, but more would like this for my own use.  Probably very minimal user journey

- App to (un)mark as taken
- Watchface complication to show medicine to take
- Tile to show medicine and mark

**Comment by waseefakhtar** [comment]
Definitely! I had it planned to work on it at some point, so I'd happily accept it as a contribution too.

The minimal user journey sounds good to start with!

**Comment by HammyHavoc** [comment]
Would be eternally grateful if this was a thing. It would make staying on top of medication for me and family members so much easier. Even MediSafe can't do this, it only has notifications.
